daemon/container_operations_unix.go

```diff
@@ -1,4 +1,4 @@
-// +build linux freebsd darwin
+// +build linux freebsd
 
```

Comment:
instead of having new _darwin.go file, decoupling the function of errors to _linux.go and keep as-is would be nicer.